packages/create-catalyst/src/utils/check-storefront-limit.ts

```diff
@@ -0,0 +1,30 @@
+import chalk from 'chalk';
+
+import { Https } from './https';
+
+export const checkStorefrontLimit = async (bc: Https) => {
```

Comment:
Going to see if I can refactor this a bit to include unit tests with this PR. I'm considering either:
* Passing `bigcommerceApiUrl`, `storeHash`, and `accessToken` as params, creating the client inside this function, and then using `msw` to mock responses
* Or, passing `availableChannels` and `features` into this method, using the types for these responses from the `zod` objects in `https.ts` (so they're always in sync). The benefit here is that we can reuse `availableChannels` in the "use existing channel" prompt in `create.ts`.

Reply:
✅